src/main/webapp/main.js

```diff
@@ -69,6 +69,8 @@ function init() {
 				
 				window.history.pushState("state", "", window.location.href.substring(0, window.location.href.indexOf("?"))); //trim off variable
 				
+				toggleCreator();
+				
 				document.getElementById('p').innerHTML = "Loaded image set.";
```

Comment:
Everything looks good except sometimes I see "Loaded image set." when I'd rather now.  Specifically if you click "View image sets of this bag." then choose one, it appears below the image set (which is fine, but perhaps not necessary).  When I click "Hide Image Set Creator" it's still there on the right.

I think the best approach is to never show that message.  The "Working..." message is useful because I'm expecting an image set to show up, but once it's there, the "Loaded image set." message is redundant.